framework-plugins/lisk-framework-monitor-plugin/src/monitor_plugin.ts

```diff
@@ -197,7 +197,7 @@ export class MonitorPlugin extends BasePlugin {
 		if (this._state.forks.blockHeaders[blockId]) {
 			this._state.forks.blockHeaders[blockId].timeReceived = Date.now();
 		} else {
-			const decodedHeader = codec.decode<RawBlockHeader>(this.schemas.blockHeader, header);
+			const decodedHeader = codec.decodeJSON<RawBlockHeader>(this.schemas.blockHeader, header);
```

Comment:
```suggestion
			const decodedHeader = codec.decodeJSON<object>(this.schemas.blockHeader, header);
```

Reply:
prefer to use `Record<string, unknown>` in case of JSON object